Tiltfile

```diff
@@ -0,0 +1,206 @@
+# -*- mode: Python -*-
+
+k8s_yaml([
+'config/crd/bases/velero.io_backups.yaml',
+'config/crd/bases/velero.io_backupstoragelocations.yaml', 
+'config/crd/bases/velero.io_deletebackuprequests.yaml',   
+'config/crd/bases/velero.io_downloadrequests.yaml',   
+'config/crd/bases/velero.io_podvolumebackups.yaml',  
+'config/crd/bases/velero.io_podvolumerestores.yaml',  
+'config/crd/bases/velero.io_resticrepositories.yaml',
+'config/crd/bases/velero.io_restores.yaml',    
+'config/crd/bases/velero.io_schedules.yaml', 
+'config/crd/bases/velero.io_serverstatusrequests.yaml',  
+'config/crd/bases/velero.io_volumesnapshotlocations.yaml', 
+])
+
+# default values
+settings = {
+   "default_registry": "",
+   "enable_restic": False,
+   "create_backup_locations": False, 
+   "setup-minio": False, 
+}
+
+# global settings
+settings.update(read_json(
+    "tilt-resources/tilt-settings.json",
+    default = {},
+))
+
+k8s_yaml(kustomize('tilt-resources'))
+k8s_yaml('tilt-resources/deployment.yaml')
+if settings.get("enable_restic"):
+    k8s_yaml('tilt-resources/restic.yaml')
+if settings.get("create_backup_locations"):
+    k8s_yaml('tilt-resources/velero_v1_backupstoragelocation.yaml')
+if settings.get("setup-minio"):
+    k8s_yaml('examples/minio/00-minio-deployment.yaml', allow_duplicates=True)
+
+# By default, Tilt automatically allows Minikube, Docker for Desktop, Microk8s, Red Hat CodeReady Containers, Kind, K3D, and Krucible.
+allow_k8s_contexts(settings.get("allowed_contexts"))
+default_registry(settings.get("default_registry"))
+local_goos = str(local("go env GOOS", quiet=True, echo_off=True)).strip()
+
+tilt_helper_dockerfile_header = """
+# Tilt image
+FROM golang:1.15.3 as tilt-helper
+# Support live reloading with Tilt
+RUN wget --output-document /restart.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/restart.sh  && \
+    wget --output-document /start.sh --quiet https://raw.githubusercontent.com/windmilleng/rerun-process-wrapper/master/start.sh && \
+    chmod +x /start.sh && chmod +x /restart.sh
+"""
+
+additional_docker_helper_commands = """
+RUN wget -qO- https://dl.k8s.io/v1.19.2/kubernetes-client-linux-amd64.tar.gz | tar xvz
+RUN wget -qO- https://get.docker.com | sh
+"""
+
+additional_docker_build_commands = """
+COPY --from=tilt-helper /usr/bin/docker /usr/bin/docker
+COPY --from=tilt-helper /go/kubernetes/client/bin/kubectl /usr/bin/kubectl
+"""
+
+##############################
+# Setup Velero
+##############################
+
+# Set up a local_resource build of the Velero binary. The binary is written to _tiltbuild/velero.
+local_resource(
+    "velero_server_binary",
+    cmd = 'cd ' + '.' + ';mkdir -p _tiltbuild;PKG=. BIN=velero GOOS=linux GOARCH=amd64 VERSION=main GIT_TREE_STATE=dirty OUTPUT_DIR=_tiltbuild ./hack/build.sh',
```

Comment:
this looks like it is running the same command run by a make target.
Any reason not to call the make target directly?

Reply:
we should also be building the `restic-restore-helper`.

Reply:
We could do this now but the code in this image looks like it hasn't changed since it was added with the initial restic support so it doesn't seem like it would be needed as urgently as the images already included in this change.

Also, I don't quite know what it means to update images used as an init container like this one. Would it just be a matter of rebuilding the image and tilt would know to use that image when it used?

Reply:
It's not obvious to me what to do with this and I have to figure this out but would like not to do it in this PR. I opened an issue to track it: https://github.com/vmware-tanzu/velero/issues/3132.